src/idl_gen_general.cpp

```diff
@@ -1146,11 +1147,18 @@ static bool SaveClass(const LanguageParameters &lang, const Parser &parser,
   return SaveFile(filename.c_str(), code, false);
 }
 
-bool GenerateGeneral(const Parser &parser,
-                     const std::string &path,
-                     const std::string & file_name) {
-
-  assert(parser.opts.lang <= IDLOptions::kMAX);
+/** it'll be split later in java/csharp... and moved to separate files */
```

Comment:
don't put these kinds of explanations in code.. put them in the commit message or discuss them elsewhere